### PR TITLE
Updated instructions to configure okta auth

### PR DESCRIPTION
**Pull request by Mkrummz**
updated configuration for connecting grafana to okta

see video walkthrough here: https://grafana.zoom.us/rec/share/nz0lj3QLSXsLqQNUM9X36AnhzuiNHCNlWFW-6eNYnSfgfI2UX6yDNMg7l6rtnPqK.LP7lvQSC3epJcTQN 
Passcode: %0$P%vH1

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md [hunk 1]
```diff
@@ -8,61 +8,100 @@ weight: 1200
 
 # Configure Okta OAuth2 authentication
 
-> Only available in Grafana v7.0+
+The Okta authentication enables Grafana users to sign in by using an external Okta authorization server.
 
-The Okta authentication allows your Grafana users to log in by using an external Okta authorization server.
+## Before you begin
+
+To complete this guide, you need:
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md:15]: @Mkrummz , this is the place where we can add prerequisites such as permissions the user must have or any knowledge we assume that they have. Do we have any such before you begin items to add?

File: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md [hunk 2]
```diff
@@ -8,61 +8,100 @@ weight: 1200
...
+   | Sign-in redirect URIs             | If you have a Grafana Cloud account, enter `https://<org>.grafana.net/login/okta`. If you self-host Grafana, enter `https://<base grafana url>/login/okta`. |
+   | Sign-out redirect URIs (Optional) | If you have a Grafana Cloud account, enter `https://<org>.grafana.net/login/okta`. If you self-host Grafana, enter `https://<base grafana url>/login/okta`. |
+   | Base URIs (Optional)              | If you have a Grafana Cloud account, enter `https://<org>.grafana.net`. If you self-host Grafana, enter `https://<base grafana url>`.                       |
+   | Controlled access                 | Select whether to assign the app integration to everyone in your org, only selected group(s), or skip the assignment until after app creation.              |
 
-1. Enter values for the **Login redirect URI**. Use **Base URI** and append it with `/login/okta`, for example: https://grafana.example.com/login/okta.
+1. Click **Done**.
 
-1. Click **Done** to finish creating the Okta application.
+## Enable Okta OAuth in Grafana Instance
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md:50]:
```suggestion
## Enable Okta OAuth in a Grafana instance
```

File: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md [hunk 3]
```diff
@@ -8,61 +8,100 @@ weight: 1200
...
 
-## Enable Okta OAuth in Grafana
+This section includes tasks you must complete to enable Okta OAuth in self-hosted Grafana or Grafana Cloud.
 
-1. Add the following to the [Grafana configuration file]({{< relref "../../../configure-grafana/#config-file-locations" >}}):
+### Enable Okta OAuth in self-hosted Grafana
+
+If you are hosting your own Grafana instance, ensure it is [accessed over HTTPS]({{< relref "../../../set-up-https" >}}). Okta requires secure communcation for exchanging authentication information between the browser and the idenitity provider.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md:56]: @Mkrummz , The HTTPS docs are a brand-new addition to the docs! Can you please verify that these are the docs we want to send users to?

**Comment by chri2547** [comment]
@Mkrummz , Hi Maggie, I've provided some feedback to you. Would you be able to address? Thank you.

**Comment by GrafanaWriter** [comment]
Moving to Blocked status since we aren't getting a response to the PR review.

**Pull request closed without merging**